packages/cli/src/commands/setup/auth/providers/nhost.js

```diff
@@ -0,0 +1,23 @@
+// the lines that need to be added to index.js
+export const config = {
+  imports: [`import { createClient } from 'nhost-js-sdk'`],
+  init: `const nhostClient = createClient({
+  baseURL: process.env.NHOST_BACKEND_URL,
+  autoLogin: false,
+});
+`,
+  authProvider: {
+    client: 'nhostClient',
+    type: 'nhost',
+    skipFetchCurrentUser: true,
```

Comment:
I am not sure why, but this property is not being included in the generated auth component. I will look into this again soon.

Reply:
`skipFetchCurrentUser` is not supported here. Only `client`, `type` and `render`
Is it important for your auth implementation to always have this set to `true`?

Reply:
@Tobbe I see, I assumed this would be picked since it is mentioned somewhere in the docs that it is a valid option for the auth provider. It doesn't hurt, but it's a failed request if one is not using prisma/disables default API.

Reply:
Yeah, it's a valid option, but the user has to set it themselves for now.

Reply:
Maybe add something about this in the docs (that also needs to be written). Like a recommendation to set it, and why it's recommended.

Reply:
Will resolve this here since we are discussing it on the docs PR here https://github.com/redwoodjs/redwoodjs.com/pull/562